docs/sources/tempo/getting-started/metrics-from-traces.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Metrics from traces
+description: Learn about how we can correlate traces and metrics.
+weight: 700
+---
+
+# Metrics from traces
+
```

Comment:
We're missing an introduction here about metrids-generator and what metrics from traces are. I'm borrowing some of the text from the metrics-generator page here. What do you think? 

```suggestion
Tempo can generate metrics from ingested traces using the metrics-generator, an optional Tempo component. The metrics-generator runs two different processors: service graphs and span metrics. 
[Service graphs]({{< relref "../metrics-generator/service_graphs#service-graphs" >}}) (generated by the processor of the same name) represent the relationships between services within a distributed system.
The [span metrics processor]({{< relref "../metrics-generator/span_metrics/" >}}) generates metrics from ingested tracing data, including request, error, and duration (RED) metrics.

```

Reply:
I like this. It's concise and neatly refers to the processors' own docs for further reading.

> We want the docs to be accurate, accessible, and inviting. "Look at these cool things you can do with Tempo! See? It's (relatively) easy."

I think it really brings your point forward. Having this information accessible at-a-glance is great and if I were a user landing on this page for the first time, I'd have all the relevant information I need.